lib/orogen/gen/typekit.rb

```diff
@@ -1994,7 +1994,11 @@ def generate
 
                 # Generate opaque-related stuff first, so that we see them in
                 # the rest of the typelib-registry-manipulation code
+                #
+                # First make sure we've loaded everything
+                perform_pending_loads
                 handle_opaques_generation(registry)
+                # All do loads another type for the m-types
```

Comment:
that sentence is somewhat unclear

Reply:
"somewhat" ... the understatement of the month ...